daemon/api_test.go

```diff
@@ -1418,7 +1418,7 @@ func (s *apiSuite) TestFind(c *check.C) {
 	c.Assert(snaps, check.HasLen, 1)
 	c.Assert(snaps[0]["name"], check.Equals, "store")
 	c.Check(snaps[0]["prices"], check.IsNil)
-	c.Check(snaps[0]["screenshots"], check.HasLen, 0)
+	c.Check(snaps[0]["screenshots"], check.IsNil)
```

Comment:
That's interesting. I would suspect the checker to behave the same for `var foo []string` and `foo := []string{}`. Both have `len(..) == 0`.

Reply:
Yes, I was thinking the same thing.